lib/nexus_cli/tasks.rb

```diff
@@ -5,6 +5,7 @@ module NexusCli
   module Tasks
     def self.included(base)
       base.send :include, ::Thor::Actions
+      # noinspection RubyInstanceMethodNamingConvention
```

Comment:
These are injected by rubymine (or intellij) to tell it to ignore style guide violations, like too long class or method names. I don't know if other analyzers support them. I guess I should take them out :)